docs/modeling/compound-models.rst

```diff
@@ -272,22 +272,22 @@ It is also possible, and sometimes useful, to make a compound model from a
 combination of classes *and* instances in the same expression::
 
     >>> from astropy.modeling.models import Linear1D, Sine1D
-    >>> MyModel = Linear1D + Sine1D(amplitude=1, frequency=1)
+    >>> MyModel = Linear1D + Sine1D(amplitude=1, frequency=1, phase=0)
     >>> MyModel
     <class '__main__.CompoundModel...'>
     Name: CompoundModel...
     Inputs: ('x',)
     Outputs: ('y',)
-    Fittable parameters: ('slope_0', 'intercept_0', 'amplitude_1', 'frequency_1')
+    Fittable parameters: ('slope_0', 'intercept_0', 'amplitude_1', 'frequency_1', 'phase_1')
     Expression: [0] + [1]
     Components:
         [0]: <class 'astropy.modeling.functional_models.Linear1D'>
         Name: Linear1D
         Inputs: ('x',)
         Outputs: ('y',)
-        Fittable parameters: ('slope', 'intercept')
+        Fittable parameters: ('slope', 'intercept', 'phase')
```

Comment:
You accidentally added phase here too.  This is a list of the individual components of a compound model, so the "Fittable parameters" above are the parameters of the full compound model.  But these are the parameters of the first component of the model which is a `Linear1D`.